driver-core/src/test/java/com/datastax/driver/core/MockClocks.java

```diff
@@ -21,7 +21,7 @@
         int calls;
 
         @Override
-        public long currentTime() {
+        public long currentTimeMicros() {
```

Comment:
Unfortunately that would not work here, as this method is inherited from Clock interface, so it needs to be public.